libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/TunnelEntranceAlert.kt

```diff
@@ -5,13 +5,15 @@ import com.mapbox.geojson.Point
 /**
  * Route alert type that provides information about tunnels on the route.
  *
+ * @param info tunnel information
  * @see RouteAlert
  * @see RouteAlertType.TunnelEntrance
  */
 class TunnelEntranceAlert private constructor(
     coordinate: Point,
     distance: Double,
-    alertGeometry: RouteAlertGeometry?
+    alertGeometry: RouteAlertGeometry?,
+    val info: TunnelInfo?
```

Comment:
```suggestion
    info: TunnelInfo?
```

nit: just consistency one way or another

Reply:
Nah the getter is needed actually.

Reply:
hmm ok, interesting the others don't need getters?

Reply:
oh they're all part of base `RouteAlert`